README.md

```diff
@@ -29,7 +29,7 @@ Table of Contents
 
 ## Requirements
 - Ruby > 2.0
-- Doorkeeper > 1.4.0 and < 4.0
+- Doorkeeper > 1.4.0 and <= 4.0
```

Comment:
Let's be not so optimistic - < 4.1 will almost certainly work without problems, while further upgrade will need to be tested prior of announcing its support :)